Lombiq.DataTables/Services/ExcelDataTableExportService.cs

```diff
@@ -67,6 +67,14 @@ public static Stream CollectionToStream(
         string error = null,
         IDictionary<int, string> customNumberFormat = null)
     {
+        // The worksheet name must be a non-empty string no longer than 31 characters. If you give it a longer name in
+        // Microsoft Excel or LibreOffice Calc and then save it in the XLSX (Excel 2007-365) format, the result will
+        // also be truncated when reopened so this is an authentic behavior.
+        worksheetName = worksheetName?.Trim();
+        worksheetName = string.IsNullOrEmpty(worksheetName)
+            ? localizer["Untitled Worksheet"]
```

Comment:
This is the Excel default:

```suggestion
            ? localizer["Sheet1"]
```